testing/testing_app.go

```diff
@@ -60,8 +60,12 @@ func SetupTestingApp() (TestingApp, map[string]json.RawMessage) {
 // of one consensus engine unit (10^6) in the default token of the simapp from first genesis
 // account. A Nop logger is set in SimApp.
 func SetupWithGenesisValSet(tb testing.TB, valSet *cmttypes.ValidatorSet, genAccs []authtypes.GenesisAccount, chainID string, powerReduction sdkmath.Int, balances ...banktypes.Balance) TestingApp {
```

Comment:
note: this function is no longer used, but left alone to prevent breaking the api

Reply:
Seems OK!